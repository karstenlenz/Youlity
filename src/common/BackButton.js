import React from 'react'
import { useHistory } from 'react-router-dom'
import styled from 'styled-components/macro'
import Button from './Button'

export default function BackButton() {
  const history = useHistory()
  return (
    <ButtonContainer>
      <Button btnType="secondary" onClick={() => history.goBack()}>
        zur√ºck
      </Button>
    </ButtonContainer>
  )
}

const ButtonContainer = styled.div`
  position: fixed;
  left: 15px;
  right: 15px;
  bottom: 0;
  z-index: var(--floating-button);
`
