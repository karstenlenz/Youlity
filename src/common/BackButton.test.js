import { render } from '@testing-library/react'
import React from 'react'
import BackButton from './BackButton'

describe('The BackButton component', () => {
  it('renders correctly', () => {
    const { container } = render(<BackButton />)
    expect(container).toMatchSnapshot()
  })
})
