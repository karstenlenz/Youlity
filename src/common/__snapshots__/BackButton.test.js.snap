// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The BackButton component renders correctly 1`] = `
<div>
  <div
    class="BackButton__ButtonContainer-xw370m-0 bZdOVN"
  >
    <button
      class="Button__BtnStyled-sc-129yfz5-0 iScSVT"
      width="100"
    >
      zur√ºck
    </button>
  </div>
</div>
`;
